[2017-06-28 15:11:59] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:11:59] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:11:59] app.CRITICAL: RuntimeException: Controller "Closure" requires that you provide a value for the "$request" argument. Either the argument is nullable and no null value has been provided, no default value has been provided or because there is a non optional argument after this one. (uncaught exception) at /myApp/vendor/symfony/http-kernel/Controller/ArgumentResolver.php line 78 {"exception":"[object] (RuntimeException(code: 0): Controller \"Closure\" requires that you provide a value for the \"$request\" argument. Either the argument is nullable and no null value has been provided, no default value has been provided or because there is a non optional argument after this one. at /myApp/vendor/symfony/http-kernel/Controller/ArgumentResolver.php:78)"} []
[2017-06-28 15:12:43] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:12:43] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:12:43] app.CRITICAL: RuntimeException: Controller "Closure" requires that you provide a value for the "$request" argument. Either the argument is nullable and no null value has been provided, no default value has been provided or because there is a non optional argument after this one. (uncaught exception) at /myApp/vendor/symfony/http-kernel/Controller/ArgumentResolver.php line 78 {"exception":"[object] (RuntimeException(code: 0): Controller \"Closure\" requires that you provide a value for the \"$request\" argument. Either the argument is nullable and no null value has been provided, no default value has been provided or because there is a non optional argument after this one. at /myApp/vendor/symfony/http-kernel/Controller/ArgumentResolver.php:78)"} []
[2017-06-28 15:13:04] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:13:04] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:13:04] app.CRITICAL: RuntimeException: Controller "Closure" requires that you provide a value for the "$request" argument. Either the argument is nullable and no null value has been provided, no default value has been provided or because there is a non optional argument after this one. (uncaught exception) at /myApp/vendor/symfony/http-kernel/Controller/ArgumentResolver.php line 78 {"exception":"[object] (RuntimeException(code: 0): Controller \"Closure\" requires that you provide a value for the \"$request\" argument. Either the argument is nullable and no null value has been provided, no default value has been provided or because there is a non optional argument after this one. at /myApp/vendor/symfony/http-kernel/Controller/ArgumentResolver.php:78)"} []
[2017-06-28 15:13:22] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:13:22] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:13:22] app.CRITICAL: RuntimeException: Controller "Closure" requires that you provide a value for the "$request" argument. Either the argument is nullable and no null value has been provided, no default value has been provided or because there is a non optional argument after this one. (uncaught exception) at /myApp/vendor/symfony/http-kernel/Controller/ArgumentResolver.php line 78 {"exception":"[object] (RuntimeException(code: 0): Controller \"Closure\" requires that you provide a value for the \"$request\" argument. Either the argument is nullable and no null value has been provided, no default value has been provided or because there is a non optional argument after this one. at /myApp/vendor/symfony/http-kernel/Controller/ArgumentResolver.php:78)"} []
[2017-06-28 15:14:02] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:14:02] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:14:23] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:14:23] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:14:23] app.NOTICE: E_NOTICE: Undefined property: Symfony\Component\HttpFoundation\ParameterBag::$address {"code":8,"message":"Undefined property: Symfony\\Component\\HttpFoundation\\ParameterBag::$address","file":"/myApp/web/HeliosAPIControllerProvider.php","line":195} []
[2017-06-28 15:14:23] app.CRITICAL: PHPUnit_Framework_Error_Notice: Undefined property: Symfony\Component\HttpFoundation\ParameterBag::$address (uncaught exception) at /myApp/web/HeliosAPIControllerProvider.php line 195 {"exception":"[object] (PHPUnit_Framework_Error_Notice(code: 8): Undefined property: Symfony\\Component\\HttpFoundation\\ParameterBag::$address at /myApp/web/HeliosAPIControllerProvider.php:195)"} []
[2017-06-28 15:15:50] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:15:50] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:15:50] app.NOTICE: E_NOTICE: Undefined property: Symfony\Component\HttpFoundation\ParameterBag::$address {"code":8,"message":"Undefined property: Symfony\\Component\\HttpFoundation\\ParameterBag::$address","file":"/myApp/web/HeliosAPIControllerProvider.php","line":195} []
[2017-06-28 15:15:50] app.CRITICAL: PHPUnit_Framework_Error_Notice: Undefined property: Symfony\Component\HttpFoundation\ParameterBag::$address (uncaught exception) at /myApp/web/HeliosAPIControllerProvider.php line 195 {"exception":"[object] (PHPUnit_Framework_Error_Notice(code: 8): Undefined property: Symfony\\Component\\HttpFoundation\\ParameterBag::$address at /myApp/web/HeliosAPIControllerProvider.php:195)"} []
[2017-06-28 15:16:12] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:16:12] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:16:12] app.NOTICE: E_NOTICE: Undefined property: Symfony\Component\HttpFoundation\ParameterBag::$address {"code":8,"message":"Undefined property: Symfony\\Component\\HttpFoundation\\ParameterBag::$address","file":"/myApp/web/HeliosAPIControllerProvider.php","line":195} []
[2017-06-28 15:16:12] app.CRITICAL: PHPUnit_Framework_Error_Notice: Undefined property: Symfony\Component\HttpFoundation\ParameterBag::$address (uncaught exception) at /myApp/web/HeliosAPIControllerProvider.php line 195 {"exception":"[object] (PHPUnit_Framework_Error_Notice(code: 8): Undefined property: Symfony\\Component\\HttpFoundation\\ParameterBag::$address at /myApp/web/HeliosAPIControllerProvider.php:195)"} []
[2017-06-28 15:16:36] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:16:36] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:16:36] app.NOTICE: E_NOTICE: Undefined property: Symfony\Component\HttpFoundation\ParameterBag::$address {"code":8,"message":"Undefined property: Symfony\\Component\\HttpFoundation\\ParameterBag::$address","file":"/myApp/web/HeliosAPIControllerProvider.php","line":195} []
[2017-06-28 15:16:36] app.CRITICAL: PHPUnit_Framework_Error_Notice: Undefined property: Symfony\Component\HttpFoundation\ParameterBag::$address (uncaught exception) at /myApp/web/HeliosAPIControllerProvider.php line 195 {"exception":"[object] (PHPUnit_Framework_Error_Notice(code: 8): Undefined property: Symfony\\Component\\HttpFoundation\\ParameterBag::$address at /myApp/web/HeliosAPIControllerProvider.php:195)"} []
[2017-06-28 15:17:01] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:17:01] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:17:01] app.ERROR: Symfony\Component\HttpKernel\Exception\HttpException: Invalid request. (uncaught exception) at /myApp/vendor/silex/silex/src/Silex/Application.php line 316 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\HttpException(code: 0): Invalid request. at /myApp/vendor/silex/silex/src/Silex/Application.php:316)"} []
[2017-06-28 15:17:25] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:17:25] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:17:25] app.ERROR: Symfony\Component\HttpKernel\Exception\HttpException: Invalid request2. (uncaught exception) at /myApp/vendor/silex/silex/src/Silex/Application.php line 316 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\HttpException(code: 0): Invalid request2. at /myApp/vendor/silex/silex/src/Silex/Application.php:316)"} []
[2017-06-28 15:18:17] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:18:17] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:18:17] app.ERROR: Symfony\Component\HttpKernel\Exception\HttpException: Invalid request. (uncaught exception) at /myApp/vendor/silex/silex/src/Silex/Application.php line 316 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\HttpException(code: 0): Invalid request. at /myApp/vendor/silex/silex/src/Silex/Application.php:316)"} []
[2017-06-28 15:20:47] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:20:47] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:20:47] app.ERROR: Symfony\Component\HttpKernel\Exception\HttpException: Invalid request. (uncaught exception) at /myApp/vendor/silex/silex/src/Silex/Application.php line 316 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\HttpException(code: 0): Invalid request. at /myApp/vendor/silex/silex/src/Silex/Application.php:316)"} []
[2017-06-28 15:21:02] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:21:02] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:21:02] app.ERROR: Symfony\Component\HttpKernel\Exception\HttpException: Invalid request1. (uncaught exception) at /myApp/vendor/silex/silex/src/Silex/Application.php line 316 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\HttpException(code: 0): Invalid request1. at /myApp/vendor/silex/silex/src/Silex/Application.php:316)"} []
[2017-06-28 15:21:59] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:21:59] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:21:59] app.ERROR: Symfony\Component\HttpKernel\Exception\HttpException: Invalid request. (uncaught exception) at /myApp/vendor/silex/silex/src/Silex/Application.php line 316 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\HttpException(code: 0): Invalid request. at /myApp/vendor/silex/silex/src/Silex/Application.php:316)"} []
[2017-06-28 15:22:54] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:22:54] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:23:18] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:23:18] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:23:19] app.CRITICAL: Doctrine\DBAL\DBALException: An exception occurred while executing 'INSERT INTO TabCisOrg (CisloOrg, Nazev, DruhyNazev, Ulice, OrCislo, PopCislo, Misto, Stav, NadrizenaOrg, PSC, Kontakt, ICO, DIC) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [999999001, "Jmeno klienta", "Druhe jmeno klienta", "Ulice", "10", "20", "Mesto", 0, null, "12345", "Kontakt", "ic", "dic"]:  SQLSTATE[42000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]99996|63877 di dic (uncaught exception) at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php line 131 {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occurred while executing 'INSERT INTO TabCisOrg (CisloOrg, Nazev, DruhyNazev, Ulice, OrCislo, PopCislo, Misto, Stav, NadrizenaOrg, PSC, Kontakt, ICO, DIC) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [999999001, \"Jmeno klienta\", \"Druhe jmeno klienta\", \"Ulice\", \"10\", \"20\", \"Mesto\", 0, null, \"12345\", \"Kontakt\", \"ic\", \"dic\"]:\n\nSQLSTATE[42000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]99996|63877\rdi\rdic at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php:131, Doctrine\\DBAL\\Driver\\PDOException(code: 42000): SQLSTATE[42000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]99996|63877\rdi\rdic at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:93, PDOException(code: 42000): SQLSTATE[42000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]99996|63877\rdi\rdic at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:91)"} []
[2017-06-28 15:23:23] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:23:23] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:23:23] app.CRITICAL: Doctrine\DBAL\DBALException: An exception occurred while executing 'INSERT INTO TabCisOrg (CisloOrg, Nazev, DruhyNazev, Ulice, OrCislo, PopCislo, Misto, Stav, NadrizenaOrg, PSC, Kontakt, ICO, DIC) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [999999001, "Jmeno klienta", "Druhe jmeno klienta", "Ulice", "10", "20", "Mesto", 0, null, "12345", "Kontakt", "ic", "dic"]:  SQLSTATE[42000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]99996|63877 di dic (uncaught exception) at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php line 131 {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occurred while executing 'INSERT INTO TabCisOrg (CisloOrg, Nazev, DruhyNazev, Ulice, OrCislo, PopCislo, Misto, Stav, NadrizenaOrg, PSC, Kontakt, ICO, DIC) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [999999001, \"Jmeno klienta\", \"Druhe jmeno klienta\", \"Ulice\", \"10\", \"20\", \"Mesto\", 0, null, \"12345\", \"Kontakt\", \"ic\", \"dic\"]:\n\nSQLSTATE[42000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]99996|63877\rdi\rdic at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php:131, Doctrine\\DBAL\\Driver\\PDOException(code: 42000): SQLSTATE[42000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]99996|63877\rdi\rdic at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:93, PDOException(code: 42000): SQLSTATE[42000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]99996|63877\rdi\rdic at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:91)"} []
[2017-06-28 15:23:50] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:23:50] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:24:00] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:24:00] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:24:09] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:24:09] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:24:16] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:24:16] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:24:23] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:24:23] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:24:31] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:24:31] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:24:44] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:24:44] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:24:44] app.ERROR: Symfony\Component\HttpKernel\Exception\HttpException: Invalid request. (uncaught exception) at /myApp/vendor/silex/silex/src/Silex/Application.php line 316 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\HttpException(code: 0): Invalid request. at /myApp/vendor/silex/silex/src/Silex/Application.php:316)"} []
[2017-06-28 15:25:26] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:25:26] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:25:26] app.ERROR: Symfony\Component\HttpKernel\Exception\HttpException: Invalid request. (uncaught exception) at /myApp/vendor/silex/silex/src/Silex/Application.php line 316 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\HttpException(code: 0): Invalid request. at /myApp/vendor/silex/silex/src/Silex/Application.php:316)"} []
[2017-06-28 15:25:36] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:25:36] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:26:07] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:26:07] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:26:28] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:26:28] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:26:50] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:26:50] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:26:56] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:26:56] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:26:56] app.CRITICAL: Doctrine\DBAL\DBALException: An exception occurred while executing 'INSERT INTO TabCisOrg (CisloOrg, Nazev, DruhyNazev, Ulice, OrCislo, PopCislo, Misto, Stav, NadrizenaOrg, PSC, Kontakt, ICO, DIC) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [999999001, "Jmeno klienta", "Druhe jmeno klienta", "Ulice", "10", "20", "Mesto", 0, null, "12345", "Kontakt", "ic", "dic"]:  SQLSTATE[42000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]99996|63877 di dic (uncaught exception) at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php line 131 {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occurred while executing 'INSERT INTO TabCisOrg (CisloOrg, Nazev, DruhyNazev, Ulice, OrCislo, PopCislo, Misto, Stav, NadrizenaOrg, PSC, Kontakt, ICO, DIC) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [999999001, \"Jmeno klienta\", \"Druhe jmeno klienta\", \"Ulice\", \"10\", \"20\", \"Mesto\", 0, null, \"12345\", \"Kontakt\", \"ic\", \"dic\"]:\n\nSQLSTATE[42000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]99996|63877\rdi\rdic at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php:131, Doctrine\\DBAL\\Driver\\PDOException(code: 42000): SQLSTATE[42000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]99996|63877\rdi\rdic at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:93, PDOException(code: 42000): SQLSTATE[42000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]99996|63877\rdi\rdic at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:91)"} []
[2017-06-28 15:53:15] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:53:15] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:53:16] app.CRITICAL: Doctrine\DBAL\DBALException: An exception occurred while executing 'INSERT INTO TabCisOrg (CisloOrg, Nazev, DruhyNazev, Ulice, OrCislo, PopCislo, Misto, Stav, NadrizenaOrg, PSC, Kontakt, ICO, DIC) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [999999001, "Jmeno klienta", "Druhe jmeno klienta", "Ulice", "10", "20", "Mesto", 0, null, "12345", "Kontakt", null, null]:  SQLSTATE[23000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]Violation of PRIMARY KEY constraint 'PK__TabCisOrg__CisloOrg'. Cannot insert duplicate key in object 'dbo.TabCisOrg'. The duplicate key value is (999999001). (uncaught exception) at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php line 131 {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occurred while executing 'INSERT INTO TabCisOrg (CisloOrg, Nazev, DruhyNazev, Ulice, OrCislo, PopCislo, Misto, Stav, NadrizenaOrg, PSC, Kontakt, ICO, DIC) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [999999001, \"Jmeno klienta\", \"Druhe jmeno klienta\", \"Ulice\", \"10\", \"20\", \"Mesto\", 0, null, \"12345\", \"Kontakt\", null, null]:\n\nSQLSTATE[23000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]Violation of PRIMARY KEY constraint 'PK__TabCisOrg__CisloOrg'. Cannot insert duplicate key in object 'dbo.TabCisOrg'. The duplicate key value is (999999001). at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php:131, Doctrine\\DBAL\\Driver\\PDOException(code: 23000): SQLSTATE[23000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]Violation of PRIMARY KEY constraint 'PK__TabCisOrg__CisloOrg'. Cannot insert duplicate key in object 'dbo.TabCisOrg'. The duplicate key value is (999999001). at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:93, PDOException(code: 23000): SQLSTATE[23000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]Violation of PRIMARY KEY constraint 'PK__TabCisOrg__CisloOrg'. Cannot insert duplicate key in object 'dbo.TabCisOrg'. The duplicate key value is (999999001). at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:91)"} []
[2017-06-28 15:56:51] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:56:51] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:56:51] app.CRITICAL: Doctrine\DBAL\ConnectionException: There is no active transaction. (uncaught exception) at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/ConnectionException.php line 42 {"exception":"[object] (Doctrine\\DBAL\\ConnectionException(code: 0): There is no active transaction. at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/ConnectionException.php:42)"} []
[2017-06-28 15:58:08] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:58:08] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:58:08] app.CRITICAL: Doctrine\DBAL\DBALException: An exception occurred while executing 'INSERT INTO TabCisOrg (CisloOrg, Nazev, DruhyNazev, Ulice, OrCislo, PopCislo, Misto, Stav, NadrizenaOrg, PSC, Kontakt, ICO, DIC) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [999999004, "Jmeno klienta", "Druhe jmeno klienta", "Ulice", "10", "20", "Mesto", 0, null, "12345", "Kontakt", null, null]:  SQLSTATE[23000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]Violation of PRIMARY KEY constraint 'PK__TabCisOrg__CisloOrg'. Cannot insert duplicate key in object 'dbo.TabCisOrg'. The duplicate key value is (999999004). (uncaught exception) at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php line 131 {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occurred while executing 'INSERT INTO TabCisOrg (CisloOrg, Nazev, DruhyNazev, Ulice, OrCislo, PopCislo, Misto, Stav, NadrizenaOrg, PSC, Kontakt, ICO, DIC) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [999999004, \"Jmeno klienta\", \"Druhe jmeno klienta\", \"Ulice\", \"10\", \"20\", \"Mesto\", 0, null, \"12345\", \"Kontakt\", null, null]:\n\nSQLSTATE[23000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]Violation of PRIMARY KEY constraint 'PK__TabCisOrg__CisloOrg'. Cannot insert duplicate key in object 'dbo.TabCisOrg'. The duplicate key value is (999999004). at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php:131, Doctrine\\DBAL\\Driver\\PDOException(code: 23000): SQLSTATE[23000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]Violation of PRIMARY KEY constraint 'PK__TabCisOrg__CisloOrg'. Cannot insert duplicate key in object 'dbo.TabCisOrg'. The duplicate key value is (999999004). at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:93, PDOException(code: 23000): SQLSTATE[23000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]Violation of PRIMARY KEY constraint 'PK__TabCisOrg__CisloOrg'. Cannot insert duplicate key in object 'dbo.TabCisOrg'. The duplicate key value is (999999004). at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:91)"} []
[2017-06-28 15:58:18] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:58:18] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:58:33] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:58:33] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 15:58:33] app.CRITICAL: Doctrine\DBAL\DBALException: An exception occurred while executing 'INSERT INTO TabCisOrg (CisloOrg, Nazev, DruhyNazev, Ulice, OrCislo, PopCislo, Misto, Stav, NadrizenaOrg, PSC, Kontakt, ICO, DIC) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [999999005, "Jmeno klienta", "Druhe jmeno klienta", "Ulice", "10", "20", "Mesto", 0, null, "12345", "Kontakt", null, null]:  SQLSTATE[23000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]Violation of PRIMARY KEY constraint 'PK__TabCisOrg__CisloOrg'. Cannot insert duplicate key in object 'dbo.TabCisOrg'. The duplicate key value is (999999005). (uncaught exception) at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php line 131 {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occurred while executing 'INSERT INTO TabCisOrg (CisloOrg, Nazev, DruhyNazev, Ulice, OrCislo, PopCislo, Misto, Stav, NadrizenaOrg, PSC, Kontakt, ICO, DIC) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [999999005, \"Jmeno klienta\", \"Druhe jmeno klienta\", \"Ulice\", \"10\", \"20\", \"Mesto\", 0, null, \"12345\", \"Kontakt\", null, null]:\n\nSQLSTATE[23000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]Violation of PRIMARY KEY constraint 'PK__TabCisOrg__CisloOrg'. Cannot insert duplicate key in object 'dbo.TabCisOrg'. The duplicate key value is (999999005). at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php:131, Doctrine\\DBAL\\Driver\\PDOException(code: 23000): SQLSTATE[23000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]Violation of PRIMARY KEY constraint 'PK__TabCisOrg__CisloOrg'. Cannot insert duplicate key in object 'dbo.TabCisOrg'. The duplicate key value is (999999005). at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:93, PDOException(code: 23000): SQLSTATE[23000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]Violation of PRIMARY KEY constraint 'PK__TabCisOrg__CisloOrg'. Cannot insert duplicate key in object 'dbo.TabCisOrg'. The duplicate key value is (999999005). at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:91)"} []
[2017-06-28 15:58:42] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 15:58:42] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 16:02:10] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 16:02:10] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 16:03:36] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 16:03:36] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 16:04:29] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 16:04:29] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 16:05:09] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 16:05:09] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 16:05:09] app.DEBUG: < 201 [] []
[2017-06-28 16:05:40] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 16:05:40] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 16:05:41] app.DEBUG: < 201 [] []
[2017-06-28 16:09:34] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 16:09:34] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 16:09:34] app.CRITICAL: Doctrine\DBAL\DBALException: An exception occurred while executing 'INSERT INTO TabCisOrg (CisloOrg, Nazev, DruhyNazev, Ulice, OrCislo, PopCislo, Misto, Stav, NadrizenaOrg, PSC, Kontakt, ICO, DIC) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [999999011, "Jmeno klienta", "Druhe jmeno klienta", "Ulice", "10", "20", "Mesto", 0, null, "12345", "Kontakt", null, null]:  SQLSTATE[23000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]Violation of PRIMARY KEY constraint 'PK__TabCisOrg__CisloOrg'. Cannot insert duplicate key in object 'dbo.TabCisOrg'. The duplicate key value is (999999011). (uncaught exception) at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php line 131 {"exception":"[object] (Doctrine\\DBAL\\DBALException(code: 0): An exception occurred while executing 'INSERT INTO TabCisOrg (CisloOrg, Nazev, DruhyNazev, Ulice, OrCislo, PopCislo, Misto, Stav, NadrizenaOrg, PSC, Kontakt, ICO, DIC) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params [999999011, \"Jmeno klienta\", \"Druhe jmeno klienta\", \"Ulice\", \"10\", \"20\", \"Mesto\", 0, null, \"12345\", \"Kontakt\", null, null]:\n\nSQLSTATE[23000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]Violation of PRIMARY KEY constraint 'PK__TabCisOrg__CisloOrg'. Cannot insert duplicate key in object 'dbo.TabCisOrg'. The duplicate key value is (999999011). at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php:131, Doctrine\\DBAL\\Driver\\PDOException(code: 23000): SQLSTATE[23000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]Violation of PRIMARY KEY constraint 'PK__TabCisOrg__CisloOrg'. Cannot insert duplicate key in object 'dbo.TabCisOrg'. The duplicate key value is (999999011). at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:93, PDOException(code: 23000): SQLSTATE[23000]: [Microsoft][ODBC Driver 13 for SQL Server][SQL Server]Violation of PRIMARY KEY constraint 'PK__TabCisOrg__CisloOrg'. Cannot insert duplicate key in object 'dbo.TabCisOrg'. The duplicate key value is (999999011). at /myApp/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:91)"} []
[2017-06-28 16:09:46] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 16:09:46] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 16:09:46] app.DEBUG: < 201 [] []
[2017-06-28 16:10:50] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 16:10:50] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 16:10:50] app.DEBUG: < 201 [] []
[2017-06-28 16:10:50] app.WARNING: E_WARNING: preg_match(): Delimiter must not be alphanumeric or backslash {"code":2,"message":"preg_match(): Delimiter must not be alphanumeric or backslash","file":"/usr/share/php/PHPUnit/Framework/Constraint/PCREMatch.php","line":48} []
[2017-06-28 16:12:13] app.INFO: Matched route "{route}". {"route":"POST_heliosapi_clients","route_parameters":{"_controller":"[object] (Closure: {})","_route":"POST_heliosapi_clients"},"request_uri":"http://localhost/heliosapi/clients","method":"POST"} []
[2017-06-28 16:12:13] app.DEBUG: > POST /heliosapi/clients [] []
[2017-06-28 16:12:13] app.DEBUG: < 201 [] []
